wrappers/base_wrapper.py

```diff
@@ -0,0 +1,44 @@
+import time
+from selenium.webdriver.remote.webelement import WebElement
+from selenium.webdriver.support.wait import WebDriverWait
+from utils import custom_expected_conditions
+
+from shishito.runtime.shishito_support import ShishitoSupport
+
+
+class BaseWrapper:
```

Comment:
!? base page base wrapper- they contains the same methods. should we  create parent object? or replace these methods to shishito?